docs/tutorial/sandbox.md

```diff
@@ -83,13 +68,21 @@ privileged APIs to untrusted code running in the renderer process unless
 
 ## Configuring the sandbox
 
-### Enabling the sandbox for a single process
+It is possible to disable the renderer sandbox on a per-process basis.
+Theoretically, unsandboxed renderers are not a problem for desktop applications that
+only display trusted code, but they make Electron less secure than Chromium for
+displaying untrusted web content. However, even purportedly trusted code may be
+dangerous — there are countless attack vectors that malicious actors can use, from
+cross-site scripting to content injection to man-in-the-middle attacks on remotely loaded
+websites, just to name a few. For this reason, we recommend enabling renderer sandboxing
+for the vast majority of cases under an abundance of caution.
```

Comment:
I think this section could be reworded to be more straightforward. There are some double-negatives here (e.g. "unsandboxed renderers are not a problem" could be reworded to "sandboxing is most relevant for ..." or so), and we can simplify the language ("purportedly", "abundance", "vast majority" are overly complex words to use here I think.)

Perhaps something like:

> For most apps, sandboxing is the best choice. In certain situations, for instance when using native node modules in the renderer, or other usages incompatible with the sandbox, it is possible to disable the sandbox for specific processes. This comes with security risks, especially if any untrusted code or content is present in the unsandboxed process.

Reply:
[8411bdd](https://github.com/electron/electron/pull/35379/commits/8411bdd40f057c8cfc16d10af326c5969c90615e)